app/services/registration_csv_converter.rb

```diff
@@ -87,6 +95,9 @@ def tags(row)
   end
 
   def dro_type(content_type)
+    # for CSV registration, we already have the URI
```

Comment:
@aaron-collier Why do we need the code below this then? Isn't this whole file for CSV registration?

Reply:
Oh you know, that's a good point @jcoyne - there was a bit of crossover in the forms, but not likely here. I'll do a follow up.

Reply:
Is it possible that content_type is included in the csv? I'll verify that first.